prestashop1.7/views/templates/admin/tawkto/helpers/view/view.tpl

```diff
@@ -456,6 +485,20 @@ jQuery(document).ready(function() {
             jQuery('.show_specific').prop('disabled', false);
         }
     });
+
+    if (jQuery('#enable_visitor_recognition').prop("checked")) {
+        jQuery('#js_api_key').prop('disabled', false);
```

Comment:
Don't disable inputs. Users may want to set up the API key before enabling visitor recognition